pyscf/adc/radc.py

```diff
@@ -2583,7 +2723,12 @@ def __init__(self, adc):
     matvec = ip_adc_matvec
     compute_trans_moments = ip_compute_trans_moments
     get_trans_moments = get_trans_moments
-    get_spec_factors = get_spec_factors_ip
+    renormalize_eigenvectors = renormalize_eigenvectors_ip
+    get_properties = get_properties
+    analyze_spec_factor = analyze_spec_factor
+    analyze_eigenvector = analyze_eigenvector_ip
+    analyze = analyze
+    compute_dyson_orb = compute_dyson_orb
```

Comment:
Please rename compute_dyson_orb to compute_dyson_mo everywhere

Reply:
Done